sql/moz-fx-data-shared-prod/search_terms/sanitization_job_languages/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+---
+friendly_name: Sanitization Job Languages
+description: |-
+  How many search terms got each language tag, by job.
+
+  We use this + other metrics to assess
+  whether the constituency using our search features is changing.
+owners:
+  - ctroy@mozilla.com
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:search-terms/aggregated
```

Comment:
We've had some discussions in the past about whether to apply access restrictions to virtual views. We could technically have this view be accessible to all of Mozilla since queries will still throw errors if the user doesn't have access to the underlying table.

But having the access enforced at the view level as well is logical and somewhat self-documenting. So I like it this way, but cc'ing @whd as to whether he has a strong opinion on it.